backend/tests/test_exceptions.py

```diff
@@ -4,7 +4,7 @@
 from rope.api.main import app
 from rope.api.sessions import get_request_session
 from rope.api.database import SessionLocal
-from rope.db.schema import UserAccount, SchoolDistrict, MoodleSetting
+from rope.db.schema import UserAccount, SchoolDistrict, MoodleSetting, CourseBuild
```

Comment:
Another "code smell" comment: This file isn't getting quite so large, but would it help navigate if we split the RBAC tests out into something like:

* `test_rbac_manager.py`
* `test_rbac_admin.py`
* `test_rbac_authenticated.py`